packages/fxa-settings/src/lib/cache.ts

```diff
@@ -3,36 +3,33 @@ import Storage from './storage';
 import { Email } from '../models';
 import { searchParam } from '../lib/utilities';
 import config from './config';
+import { StoredAccountData } from './storage-utils';
 
 const storage = Storage.factory('localStorage');
 
-export interface OldSettingsData {
-  uid: hexstring;
-  sessionToken: hexstring;
-  alertText?: string;
-  displayName?: string;
-  metricsEnabled?: boolean;
-  email?: string;
-}
-
-export function getOldSettingsData({
+export function getStoredAccountData({
```

Comment:
Filed FXA-8454 - this part of the codebase hasn't been reviewed in a few years, so it might be valuable to have a more comprehensive look at cache handling in fxa-settings.